README.md

```diff
@@ -0,0 +1,73 @@
+# Release Notes Action
+
+Generate release-notes for a given version from CHANGELOG file.
+
+## Usage
```

Comment:
I think we should add the inputs to the ##Usage section. 

If someone comes in and just copies the usage code, skips over inputs, then they will still get the arguments that the action accepts.

Reply:
I don't mind doing it, but I don't totally understand your scenario. Mind rephrasing or elaborating a little?